Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -202,7 +215,7 @@ class BackForwardListViewController: UIViewController,
             verticalConstraints += constraints
         } else {
             let statusBarHeight = UIWindow.keyWindow?.windowScene?.statusBarManager?.statusBarFrame.height ?? 0
-            tableViewTopAnchor = tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: bvc.header.frame.height + statusBarHeight)
+            tableViewTopAnchor = tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: browserFrameInfo.getHeaderSize().height + statusBarHeight) // bvc.header.frame.height + statusBarHeight)
```

Comment:
Same here, we can remove the comment
```suggestion
            tableViewTopAnchor = tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: browserFrameInfo.getHeaderSize().height + statusBarHeight)
```